File: sanidad-backend/src/main/java/com/farmacia/sanidadbackend/inteligencia/perdidas/dto/ProductoInmovilDTO.java
package com.farmacia.sanidadbackend.inteligencia.perdidas.dto;

import lombok.Data;
import java.math.BigDecimal;

@Data
public class ProductoInmovilDTO {
    private Long medicamentoId;
    private String medicamentoNombre;
    private Integer stockActual;
    private Integer ventasUltimos90Dias;
    private Integer diasSinMovimiento; // opcional
    private BigDecimal valorInmovilizado;
}

File: SanidadMovil/app/src/main/java/com/sanidad/movil/data/remote/dto/ProductoInmovilDTO.kt
package com.sanidad.movil.data.remote.dto

data class ProductoInmovilDTO(
    val medicamentoId: Long,
    val medicamentoNombre: String,
    val stockActual: Int,
    val ventasUltimos90Dias: Int,
    val diasSinMovimiento: Int,
    val valorInmovilizado: Double
)